SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```
@@ -0,0 +1,32 @@
+%define mstflint_unsigned_name mstflint-kernel
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir lib/modules/%{kver}/extra/%{name}
```

The `%{name}` macro is the value of the `Name` tag below. In this case it would translate to "mstflint-kernel-signed". I see you're storing the actual package name in `mstflint_unsigned_name`, so let's use that:
```suggestion
%global install_mod_dir lib/modules/%{kver}/extra/%{mstflint_unsigned_name}
```